roles/libvirt_manager/README.md

```diff
@@ -90,6 +90,10 @@ cifmw_libvirt_manager_configuration:
       disksize: 50
       memory: 4
       cpus: 2
+      ip_address:
```

Comment:
Shall we enable this feature for computes we well? Or, for them, there will be some DHCP anyway?

Reply:
Sounds good! I'm just "annoyed" by the hard-coded `eth0` in the script - it should be OK on the controller-0 in all (most?) of the cases, but maybe preparing a parameter may be good?